CHANGELOG.md

```diff
@@ -9,6 +9,34 @@ Add new release notes in reverse numerical order (newest first) below this comme
 You may want to add temporary notes here for tracking as features are added, before a new release is ready.
 -->
 
+## v0.2.0
+
+This release adds the first set of community-contributed analyses to the repository.
+These modules are focused on cell type identification and annotation for specific ScPCA datasets.
+Note that many of these modules are still in development at this stage, and may not yet be fully functional.
```

Comment:
Maybe remove "yet" since we don't necessarily know it will come to pass? Or, replace with "currently" or similar?
```suggestion
Note that many of these modules are still in development at this stage, and may not be fully functional.
```